gammapy/estimators/core.py

```diff
@@ -226,110 +253,151 @@ def _validate_data(data, sed_type, check_scan=False):
                 "Missing data / column for sed type '{}':" " {}".format(sed_type, missing)
             )
 
-    @property
-    def energy_axis(self):
-        """Energy axis (`MapAxis`)"""
-        return self._energy_axis
+    # TODO: add support for scan
+    def _check_quantity(self, quantity):
+        if quantity not in self.available_quantities:
+            raise AttributeError(
+                f"Cannot compute required flux quantity. {quantity} "
```

Comment:
Note that when no `counts` or no `ts` are available the message about flux quantity is strange.

Reply:
Agreed, done.